homeassistant/components/emulated_hue/__init__.py

```diff
@@ -227,6 +248,12 @@ def is_entity_exposed(self, entity):
         domain = entity.domain.lower()
         explicit_expose = entity.attributes.get(ATTR_EMULATED_HUE, None)
         explicit_hidden = entity.attributes.get(ATTR_EMULATED_HUE_HIDDEN, None)
+
+        if entity.entity_id in self.entities and \
+                CONF_ENTITY_HIDDEN in self.entities[entity.entity_id]:
+            explicit_hidden = \
+                self.entities[entity.entity_id][CONF_ENTITY_HIDDEN]
```

Comment:
I'm just a new guy here but I wonder if it's worthwhile to use [entityfilter](https://github.com/home-assistant/home-assistant/blob/dev/homeassistant/helpers/entityfilter.py) here? See alexa for [an example](https://github.com/home-assistant/home-assistant/blob/1419005082fb328b56a856aa48331e6a4cd08602/homeassistant/components/alexa/smart_home.py#L393)

Reply:
For Alexa we use both an entity config section and entity filter.